### PR TITLE
ref(dynamic-sampling): Apply minor adjusts

### PR DESCRIPTION
**Pull request by priscilawebdev**
. Add tooltip with description on Add/Drag button when disabled
. Increase Modal size
. Replaced Match TextField with a TextArea component:

**Before:**

image

smaller device:

image

**After:**

image

smaller device:

image

ui-modal-changes

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/components/forms/controls/textarea.tsx [hunk 1]
```diff
@@ -36,6 +41,7 @@ const propFilter = (p: string) =>
 
 const TextArea = styled(TextAreaControl, {shouldForwardProp: propFilter})`
   ${inputStyles};
+  padding: ${space(1)};
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/components/forms/controls/textarea.tsx:44]: this was necessary for the field to be `40px`; height

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx [hunk 2]
```diff
@@ -37,41 +39,58 @@ function ConditionFields({
...
-              hideControlState
-              showHelpInTooltip
-              required
-              stacked
-            />
+          <FieldsWrapper key={index}>
+            <Fields>
+              <SelectField
+                label={t('Category')}
+                help={t('This is a description')}
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx:50]: Is this temporary copy?

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx [hunk 3]
```diff
@@ -37,41 +39,58 @@ function ConditionFields({
...
+                choices={categoryOptions}
+                inline={false}
+                hideControlState
+                showHelpInTooltip
+                required
+                stacked
+              />
+              <StyledField
+                label={t('Match Conditions')}
+                help={t('This is a description')}
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx:63]: I would omit this so it doesn't get left behind by accident.

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx [hunk 4]
```diff
@@ -37,41 +39,58 @@ function ConditionFields({
...
+                  name={`match-${index}`}
+                  value={match}
+                  onChange={value => onChange(index, 'match', value)}
+                  disabled={showLegacyBrowsers}
+                  inline={false}
+                  autosize
+                  hideControlState
+                  stacked
+                />
+                <FieldHelp>{t('Press enter to add a new match condition')}</FieldHelp>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx:86]:
```suggestion
                <FieldHelp>{t('You can include multiple values by putting each value on a separate line')}</FieldHelp>
```
Not sure if this is better 🤷

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx:50]: yes..it's temporary 😉

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx (hunk 4 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx:86]: oh thanks! I was looking for a better phrase

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx (hunk 3 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx:63]: @jan-auer please do you already have a description for these fields?

**Review by markstory — APPROVED** [review]

**Pull request merged**